src/main/resources/static/js/itemScript.js

```diff
@@ -0,0 +1,7 @@
+
+$(function () {
+    $("#item-name").autocomplete({
+        source: ["Laptop", "Mouse Pad", "Monitor"],
+        minLength: 2
+    });
+});
```

Comment:
Same here with the JavaScript, although '<script>' tags can be included to the head tag, according to: 

[](https://htmlhead.dev/#:~:text=Valid%20elements%20include%20meta,search%20engines%2C%20bots%2C%20etc)

Business logic should not be included.

Reply:
Good to externalize JavaScript to an external .js file.